lib/slack/web/api/patches/chat.1.patch

```diff
@@ -24,18 +27,18 @@ index 54a7db1..c535bb5 100644
              post('chat.postEphemeral', options)
            end
  
-@@ -146,13 +152,19 @@ module Slack
+@@ -166,7 +177,19 @@ module Slack
            # @see https://github.com/slack-ruby/slack-api-ref/blob/master/methods/chat/chat.postMessage.json
            def chat_postMessage(options = {})
              throw ArgumentError.new('Required arguments :channel missing') if options[:channel].nil?
--            throw ArgumentError.new('Required arguments :text or :attachments missing') if options[:text].nil? && options[:attachments].nil?
+-            throw ArgumentError.new('Required arguments :text missing') if options[:text].nil?
 +            throw ArgumentError.new('Required arguments :text, :attachments or :blocks missing') if options[:text].nil? && options[:attachments].nil? && options[:blocks].nil?
-             # attachments must be passed as an encoded JSON string
-             if options.key?(:attachments)
-               attachments = options[:attachments]
-               attachments = JSON.dump(attachments) unless attachments.is_a?(String)
-               options = options.merge(attachments: attachments)
-             end
++            # attachments must be passed as an encoded JSON string
```

Comment:
Those + signs look like a patch bug, this code won't run :)

Reply:
Possibly missing something here, but isn't this the output of the diff? I was able to run the rake task fine, so I don't think it causes an issue with the patching. Example output from the [older files](https://github.com/slack-ruby/slack-ruby-client/pull/326/files#diff-75d9a357dc1666af97093cba14986108L10) also has the `+` signs.

Reply:
Bah, I don't know what I was thinking, this is a patch file. You're right.

So what happened here? Before we were removing the code that did `JSON.dump` and now we put it back? Seems like the new(er) version is correct.

Reply:
Oh looks like multiple patches got merged into one. I get it, ignore me.